scripts/pancancer_classifier.py

```diff
@@ -186,9 +186,8 @@
     mut_file = os.path.join('data', 'pancan_mutation_freeze.tsv')
     mut_burden_file = os.path.join('data', 'mutation_burden_freeze.tsv')
 elif y_matrix == 'xena':
-    mut_file = os.path.join('..', 'tybalt', 'data', 'pancan_mutation.tsv')
-    mut_burden_file = os.path.join('..', 'tybalt', 'data',
-                                   'sample_mutation_burden.tsv')
+    mut_file = 'https://media.githubusercontent.com/media/greenelab/tybalt/87496e23447a06904bf9c07c389584147b87bd65/data/pancan_mutation.tsv'
```

Comment:
I use the hyperlinks in this form:

```
https://github.com/greenelab/tybalt/raw/87496e23447a06904bf9c07c389584147b87bd65/data/pancan_mutation.tsv
```

Then all you have to do is switch `raw` to `blob` to get back to the browser view.

Semi-related: why would you not compress the 428 MB TSV file (`pancan_mutation.tsv`) that's tracked with git LFS? I believe LFS is always just storing blobs, so there is no reason not to compress?

Reply:
This is awesome, thanks!